index.bs

```diff
@@ -1360,56 +1360,176 @@ devices. For example, authors wish to be able to use the web to connect with the
-to expose the unique identifier at all. Please read the TAG's finding on
-<a href="http://www.w3.org/2001/tag/doc/unsanctioned-tracking/">unsanctioned tracking</a> for
-additional details. Despite this general concern, it may be very useful or necessary to expose a
-device's unique identifier to the web platform. The following guidelines will help ensure that this
-is done in a consistent and privacy-friendly way:
-
- : Limit identifiable information in the id
- :: As much as possible, device ids exposed to the web platform should not contain identifiable
-     information such as branding, make and model numbers, etc. In many cases using a randomized
-     number or unique id is preferable to a generic string identifier such as "device1".
- :: Device ids expressed as numbers should contain sufficient entropy so as to avoid re-use or
-     potential sharing among other devices, and should not be easily guessable.
-
- : Keep the user in control
- :: Any existing device ids mapped to or stored with the current session by the user agent should
-     be cleared when users elect to "clear their cookies" (and other related settings). Above all,
-     the user should be in control of this potential tracking state and be able to reset it on
-     demand.
-
- : Hide sensitive ids behind a user permission
- :: Where device identification does not make sense to be expressed in an anonymous way, access to
-     the identifier should be limited by default. One way to limit exposure is to only surface the
-     identifier to author code <em>after</em> obtaining permission from the user.
-
- : Tie ids to the same-origin model
- :: Identifiers should be unique to the origin of the web content that is attempting to access
-     them. Web content from one origin should receive an identifier that is distinct from the
-     identifier given to web content from any other origin despite the physical device being the
-     same.
- :: Within an origin, ids may have a stable representation to web content. This ensures a
-     consistent developer experience if requesting the same device twice.
-
- : Persistable when necessary
- :: Device identifiers obtained after a complex or time-consuming device selection process may
-     prefer that author code be allowed to persist the id for use in a later session in order to
-     avoid the selection process a second time. In this case, the API should not only provide a
-     stable id during the session for the given origin, but also be able to deterministically produce
-     the same id in subsequent sessions.
+This section contains principles for consideration when designing APIs for devices.
+
+<h3 id="device-ids">Use care when exposing identifying information about devices</h3>
+
+Exposing device identifiers
+(or information about a device
+or its capabilities
+that may be used to synthesize such an identifier)
+increases the fingerprinting surface of a user agent,
```

Comment:
This seems to imply that only information that uniquely identifies one device or device model increases the fingerprinting surface, but of course even revealing, for example, that a camera has a particular resolution will reduce the user's anonymity set.

An identifier for a particular model of device also has privacy harms beyond fingerprinting: knowing that someone has a particular speaker could help speaker-sellers target their marketing more precisely.

In the long run, we should go into more detail about this in https://w3cping.github.io/privacy-threat-model/#hl-sensitive-information...

Reply:
> This seems to imply that only information that uniquely identifies one device or device model increases the fingerprinting surface, but of course even revealing, for example, that a camera has a particular resolution will reduce the user's anonymity set.
> 
> An identifier for a particular model of device also has privacy harms beyond fingerprinting: knowing that someone has a particular speaker could help speaker-sellers target their marketing more precisely.

This is great feedback. I've tried to capture it in 2ec5b8e.

> In the long run, we should go into more detail about this in https://w3cping.github.io/privacy-threat-model/#hl-sensitive-information...

Please let us know when you think that's ready for citing!